lib/Workflow/SupportStrategy/ExpressionSupportStrategy.php

```diff
@@ -54,7 +54,8 @@ public function supports(WorkflowInterface $workflow, $subject): bool
             return false;
         }
 
-        return $this->expressionService->evaluateExpression($workflow, $subject, $this->expression);
+        $ret = $this->expressionService->evaluateExpression($workflow, $subject, $this->expression);
+        return is_bool($ret) ? (bool)$ret : false;
```

Comment:
Why cast it to bool when it already is bool?

Reply:
To be honest, since `evaluateExpression` returns mixed, I think we should just go for

`return (bool)$this->expressionService->evaluateExpression($workflow, $subject, $this->expression);`

Reply:
Or use `filter_var` instead of `is_bool`